bin/libast_prereq.sh

```diff
@@ -29,7 +29,7 @@ iffe_sh_tests=()
 
 iffe_sh_tests_2=('param.sh' )
 
-iffe_sh_tests_3=('libpath.sh' )
+iffe_sh_tests_3=('')
```

Comment:
Shouldn't this just be an empty array initializer rather than containing an empty string?

Reply:
I fixed it in a later commit.